Cargo.toml

```diff
@@ -231,3 +256,8 @@ overflow-checks = true
 
 [profile.test.package.script]
 opt-level = 2
+
+# The profile that 'cargo dist' will build with
+[profile.dist]
+inherits = "release"
+lto = "thin"
```

Comment:
We have to be careful about LTO. I do want it, but if we want it, we should test with it. There was a case when LTO lead to ruining secure memory cleanups.

Reply:
set `lto = "off"`.